hub/config/src/main/java/uk/gov/ida/hub/config/application/CertificateExpiryDateCheckService.java

```diff
@@ -13,33 +13,46 @@
 public class CertificateExpiryDateCheckService implements Runnable {
     private static final Logger LOG = LoggerFactory.getLogger(CertificateExpiryDateCheckService.class);
     private final CertificateService certificateService;
-    private final Gauge gauge;
+    private final Gauge expiryDateGauge;
+    private final Gauge lastUpdatedGauge;
 
     public CertificateExpiryDateCheckService(final CertificateService certificateService,
-                                             final Gauge gauge) {
+                                             final Gauge expiryDateGauge,
+                                             final Gauge lastUpdatedGauge) {
         this.certificateService = certificateService;
-        this.gauge = gauge;
+        this.expiryDateGauge = expiryDateGauge;
+        this.lastUpdatedGauge = lastUpdatedGauge;
```

Comment:
I don't think we need a lastUpdatedGauge for the expiry date metric. It's important for OCSP because we want to know the last time OCSP was successful, but the expiry date of a certificate isn't something that should ever change over time so we don't need to keep track of updates here.

Reply:
I understand from Andy that Verify team is planning to make changes to Config application to have a dynamic list of RP and MSA certificates during runtime (i.e. it will reload a list of RP and MSA certificates from S3 bucket or other sources every N seconds) rather than loading a list of certificates once at the beginning of the program in the near future. This is the reason why I added the timestamp to the metric.

Reply:
if that's the case, I think we only need a single last-updated gauge (for the most recent time we updated), rather than a separate gauge per certificate.  (in other words, I think we can remove all the certificate-type labels from lastUpdatedGauge)

Reply:
Done.